packages/notification-center/src/i18n/languages/az.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const AZ: ITranslationEntry = {
+  translations: {
+    notifications: 'bildirişlər',
+    markAllAsRead: 'Hamısını oxunmuş olaraq işarələ',
+    poweredBy: 'tərəfindən idarə olunur',
```

Comment:
This seems to be translated to `Managed by` rather than `Powered by`. As the notification center can be included by clients that self host their solution I find is not an appropriate translation as could mislead.

Reply:
Thanks for pointing it out… i will check and verify the same and edit it accordingly!

Reply:
> This seems to be translated to `Managed by` rather than `Powered by`. As the notification center can be included by clients that self host their solution I find is not an appropriate translation as could mislead.

I have made the changes and i found out that in Azerbaijani there is no translation for Poweredby as they you "managed by" or "supplied by".

Reply:
We'd rather go with `Supplied by` then.

Reply:
> We'd rather go with `Supplied by` then.

Yes i have used “supplied by” in the edited version!